pkg/venafi/cloud/connector.go

```diff
@@ -673,16 +672,9 @@ func normalizeURL(url string) (normalizedURL string, err error) {
 		url = apiURL
 		//return "", fmt.Errorf("base URL cannot be empty")
```

Comment:
Remove this comment as no longer needed or return right away as no further modifications are required to the default URL

Reply:
It's done